nipoppy/workflows/dataset_init.py

```diff
@@ -154,20 +155,37 @@ def _init_manifest_from_bids_dataset(self) -> None:
                     if x.is_dir() and x.name.startswith(BIDS_SESSION_PREFIX)
                 ]
             )
-            if not session_ids:
-                self.logger.warning(
-                    f"Skipping subject '{ppt}': could not find a session level folder."
-                )
-                continue
+            if len(session_ids) == 0:
```

Comment:
I know you're not the one who came up with these variable names, but for clarity I think we should do some renaming:
- `session_ids` -> `bids_session_ids`
- `ses` -> `bids_session_id`
- `participant_ids` -> `bids_participant_ids`
- `ppt` -> `bids_participant_id`

Just to make it clear that these all have the BIDS prefixes to avoid confusion in the future

Reply:
I agree!